lib/medusa/adapter/rabbitmq.ex

```diff
@@ -91,6 +95,26 @@ defmodule Medusa.Adapter.RabbitMQ do
     end
   end
 
+  def handle_call({:alive}, _from, state) do
+    # Default rabbitmq vhost
+    default_vhost = "/"
+
+    # The vhost needs to be url encoded in the path as it can contain /
+    # and it needs to be separated from the path itself.
+    # The default path is /
+    # See https://lists.rabbitmq.com/pipermail/rabbitmq-discuss/2012-March/019161.html
+    vhost = URI.encode_www_form(default_vhost)
+
+    opts = connection_opts
+    url = "http://#{opts[:username]}:#{opts[:password]}@#{opts[:host]}:15672/api/aliveness-test/#{vhost}"
```

Comment:
They are the defaults. 
Currently medusa does not support custom ports for the TCP port either, so I thought either we support all or nothing.